src/features/profile/components/EditPersonDialog/EditPersonFields.tsx

```diff
@@ -221,6 +221,38 @@ const EditPersonFields: FC<EditPersonFieldsProps> = ({
               value={fieldValues[field.slug]?.toString() ?? ''}
             />
           );
+        } else if (
+          field.type === CUSTOM_FIELD_TYPE.ENUM &&
+          field.enum_choices
+        ) {
+          return (
+            <Box alignItems="flex-start" display="flex" flex={1}>
+              <FormControl fullWidth>
+                <InputLabel>{field.title}</InputLabel>
+                <Select
+                  fullWidth
+                  label={field.title}
+                  onChange={(ev) => {
+                    let value: string | null = ev.target.value;
+                    if (value === '') {
+                      value = null;
+                    }
```

Comment:
Nice!

Reply:
Keys will never be an empty string because the API doesn't allow such keys to be configured, and  the value is always null on empty values because `selectOption` is called with `(key, null)` in `EnumConfig.tsx:82`

You're commenting old code here though?